### PR TITLE
Adds HOP beret

### PR DESCRIPTION
**Pull request by Danchi299**
## What Does This PR Do
Adds head of personnel's beret to HOP garment bag. 
Sprites included.

## Why It's Good For The Game
Most other heads have a beret that comes with their shawl, I though it would be nice if HOP also had one.
I am aware of https://github.com/ParadiseSS13/Paradise/pull/22350

## Images of changes
body
hat

## Testing
Equipped HOP stuff in Loadout.
Joined as HOP, looking dressed up, spun around, compared to rest of outfit. 
Put Beret on Vox and Drask, station did not explode.

## Changelog
:cl:
add: Added HOP beret that can be found in garment bag and loadout.
/:cl:

### CODE REVIEW

**Comment by DGamerL** [comment]
Mirroring from discord:
- The sprite having a red outline looks bad
- With Christa's PR already adding a HOP beret, this seems like needless conflicting with that PR

**Comment by Danchi299** [comment]
I'll reconsider this if https://github.com/ParadiseSS13/Paradise/pull/22350 fails

**Comment by Danchi299** [comment]
adjusted obj sprite to have better outlines
HOP - Upscale

**Comment by Danchi299** [comment]
now the only question is, should the shiny part be silver per ID and Rank or golden per other parts of outfit?

File: code/modules/clothing/head/job_hats.dm [hunk 1]
```diff
@@ -57,6 +57,11 @@
 	armor = list(MELEE = 15, BULLET = 10, LASER = 15, ENERGY = 5, BOMB = 15, RAD = 0, FIRE = 50, ACID = 50)
 	dog_fashion = /datum/dog_fashion/head/hop
 
+/obj/item/clothing/head/hopcap/beret
+	name = "head of personnel's beret"
+	icon_state = "hop_beret"
+	desc = "For doing paperwork with style."
```
Comment (Henri215) [inline code/modules/clothing/head/job_hats.dm:63]:
```suggestion
/obj/item/clothing/head/hopcap/beret
	name = "head of personnel's beret"
	desc = "For doing paperwork with style."
	icon_state = "hop_beret"
```
Style preference of having desc after the name.

**Review by Henri215 — CHANGES_REQUESTED** [review]
You need to make versions for Drask and Vox species.

**Comment by Danchi299** [comment]
right, how do I de-conflict the Image files

**Comment by Danchi299** [comment]
ok, what have I done this time

**Comment by S34NW** [comment]
image
You didn't actually do a proper merge master, you did a "fake" one

**Comment by Henri215** [comment]
image
No need for this one the changelog, when you are adding a new item its already expected it would have a sprite for him

File: code/modules/client/preference/loadout/loadout_hat.dm [hunk 2]
```diff
@@ -177,6 +177,11 @@
 	path = /obj/item/clothing/head/beret/sec
 	allowed_roles = list("Head of Security", "Warden", "Security Officer")
 
+/datum/gear/hat/beret_job/hop
+	display_name = "Beret, head of personnel's"
```
Comment (Henri215) [inline code/modules/client/preference/loadout/loadout_hat.dm:181]:
```suggestion
	display_name = "Beret, head of personnel"
```

File: code/modules/clothing/head/job_hats.dm [hunk 3]
```diff
@@ -57,6 +57,11 @@
 	armor = list(MELEE = 15, BULLET = 10, LASER = 15, ENERGY = 5, BOMB = 15, RAD = 0, FIRE = 50, ACID = 50)
 	dog_fashion = /datum/dog_fashion/head/hop
 
+/obj/item/clothing/head/hopcap/beret
+	name = "head of personnel's beret"
+	desc = "For doing paperwork with style."
+	icon_state = "hop_beret"
+
```
Comment (Henri215) [inline code/modules/clothing/head/job_hats.dm:64]:
```suggestion
	icon_state = "hop_beret"
	sprite_sheets = list(
		"Vox" = 'icons/mob/clothing/species/vox/head.dmi',
		"Drask" = 'icons/mob/clothing/species/drask/head.dmi',
		)

```
Needs these for the species variations

**Review by Henri215 — CHANGES_REQUESTED** [review]
Species sprites have no issues, nice job.

**Review by Henri215 — APPROVED** [review]

**Review by Burzah — APPROVED** [review]

**Review by S34NW — APPROVED** [review]

**Pull request merged**